lib/commands/connect/pause.js

```diff
@@ -14,19 +14,10 @@ module.exports = {
   needsApp: true,
   needsAuth: true,
   run: cli.command(co.wrap(function* (context, heroku) {
-    let connection = yield api.withConnection(context, heroku);
-    let url = '/api/v3/connections/' + connection.id + '/actions/pause';
-    let response = yield api.request(context.auth.password, 'POST', url);
-
-    switch(response.statusCode) {
-      case 202:
-        console.log(api.connection_string(connection) + ' paused');
-      break;
-      case 409:
-        console.log(response.json['message']);
-      break;
-      default:
-        cli.debug(response);
-    }
+    yield cli.action('pausing connection', co(function* () {
+      let connection = yield api.withConnection(context, heroku);
+      let url = '/api/v3/connections/' + connection.id + '/actions/pause';
+      yield api.request(context.auth.password, 'POST', url);
```

Comment:
It's just unclear to me how errors are displayed, the old way was more obvious on that front.  Dumping json doesn't seem ideal, but dumping reponse.json['message'] by default seems reasonable.